composer.json

```diff
@@ -41,8 +41,8 @@
     },
     "require-dev": {
         "bamarni/composer-bin-plugin": "^1.4.1",
-        "http-interop/http-factory-tests": "^0.7",
-        "phpunit/phpunit": "^8.5"
+        "http-interop/http-factory-tests": "^0.9",
+        "phpunit/phpunit": "^8.5.8 || ^9.3.10"
```

Comment:
Im not a big fan of this line. 

Since we are not using any features in 9.3, there is no need to install it, right?

Reply:
I think the reason for that line was it is the first version of PHPUnit that is not broken on PHP 8.0.0.

Reply:
I just think this opens the door to something like this...

https://github.com/guzzle/psr7/blob/1.x/composer.json#L24

Anyhow, lets have a proper discussion about this when someone likes to add a third major version.